scripts/flake_checker.py

```diff
@@ -150,12 +159,12 @@ def is_test_output_flaky(output_lines, suite_name):
             'Please report to E2E team in case server is down.'
             'Exception: %s') % (FLAKE_CHECK_AND_REPORT_URL, e))
 
-        return False
+        return False, RERUN_UNKNOWN
```

Comment:
Any specific reason for not using True, False, and None as rerun values?

Reply:
Just that this gives us more flexibility moving forward to issue more complex instructions, e.g. `rerun only if on circle CI`

Reply:
I see, thanks for the info!